lib/processLayerStyles.js

```diff
@@ -37,10 +37,16 @@ const processLayerStyles = async (layerStyles, filepath, outputFolder, modifiers
         // Border is aplied after the cornerRadius because we need to know the radius for the border
         if (currentModifier.modifier != null) {
           const radius = cornerRadius(currentModifier.modifier)
-          addOptionalProperty("overlay", data.style, layerBorder(layerStyle, radius))
-
-          // Clip the shape for the corner radius
-          addOptionalProperty("clipShape", data.style, cornerRadius(currentModifier.modifier) )
+          // If there is a radius we overlay it and clip it
+          if (radius === null) {
+            addOptionalProperty("overlay", data.style, layerBorder(layerStyle, radius))
+
+            // Clip the shape for the corner radius
+            addOptionalProperty("clipShape", data.style, cornerRadius(currentModifier.modifier) )
+          } else {
+            // If there is no radius or the radius is 0, then use a regular border
```

Comment:
either this comment is wrong or your check is wrong...

Reply:
Ah sure, if the radius is `0` the `cornerRadius` function returns `null`

Reply:
**BUENO**